src/translations/nl.json

```diff
@@ -87,8 +87,18 @@
     "header": "Laad of ververs gegevens",
     "explanation": "Ga naar <0>deze pagina</0> om meer gegevens in je IRMA-app te laden of om bestaande gegevens te verversen."
   },
+  "token-invalid": {
+    "header": "Er ging iets mis",
+    "explanation": "Deze link is verlopen.",
+    "explanation-details": "Je kunt alsnog je e-mail adres toevoegen zodat je jouw IRMA-app kunt blokkeren bij verlies of diefstal:",
+    "point-1":"Ga naar <0>MijnIRMA</0>.",
+    "point-2":"Klik op 'Open IRMA-app' of scan de QR code met jouw IRMA-app.",
+    "point-3":"Volg de instructies in jouw IRMA app om in te loggen in MijnIRMA.",
+    "point-4":"Klik op 'Voeg e-mail adres toe' en volg de instructies om je e-mail adres toe toe voegen aan MijnIRMA.",
```

Comment:
In het Nederlands is e-mailadres aan elkaar volgens mij.

Reply:
Done